code/game/machinery/ARES/ARES_interface_admin.dm

```diff
@@ -0,0 +1,494 @@
+
+		// -- 1:1 Conversation -- //
+		if("new_conversation")
+			var/datum/ares_record/talk_log/convo = new(interface.last_login)
+			convo.conversation += "[MAIN_AI_SYSTEM] at [worldtime2text()], 'New 1:1 link initiated. Greetings, [interface.last_login].'"
+			datacore.records_talking += convo
+
+		if("clear_conversation")
+			var/datum/ares_record/talk_log/conversation = locate(params["active_convo"])
+			if(!istype(conversation))
+				return FALSE
+			var/datum/ares_record/deleted_talk/deleted = new
+			deleted.title = conversation.title
+			deleted.conversation = conversation.conversation
+			deleted.user = MAIN_AI_SYSTEM
+			datacore.records_deletion += deleted
+			datacore.records_talking -= conversation
+
+		if("fake_message_ares")
+			var/message = tgui_input_text(user, "What do you wish to say to ARES?", "ARES Message", encode = FALSE)
+			if(message)
+				interface.message_ares(message, user, params["active_convo"], TRUE)
+		if("ares_reply")
+			var/message = tgui_input_text(user, "What do you wish to reply with?", "ARES Response", encode = FALSE)
+			if(message)
+				interface.response_from_ares(message, params["active_convo"])
+				var/datum/ares_record/talk_log/conversation = locate(params["active_convo"])
+				var/admin_log = SPAN_STAFF_IC("<b>ADMINS/MODS: [SPAN_RED("[key_name(user)] replied to [conversation.user]'s ARES message")] [SPAN_GREEN("via Remote Interface")] with: [SPAN_BLUE(message)] </b>")
+				for(var/client/admin in GLOB.admins)
+					if((R_ADMIN|R_MOD) & admin.admin_holder.rights)
+						to_chat(admin, admin_log)
+
+		if("read_record")
+			var/datum/ares_record/deleted_talk/conversation = locate(params["record"])
+			admin_interface.deleted_1to1 = conversation.conversation
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "read_deleted"
+
+		if("claim_ticket")
+			var/datum/ares_ticket/ticket = locate(params["ticket"])
+			if(!istype(ticket))
+				return FALSE
+			var/claim = TRUE
+			var/assigned = ticket.ticket_assignee
+			if(assigned)
+				if(assigned == MAIN_AI_SYSTEM)
+					var/prompt = tgui_alert(user, "ARES already claimed this ticket! Do you wish to drop the claim?", "Unclaim ticket", list("Yes", "No"))
+					if(prompt != "Yes")
+						return FALSE
+					/// set ticket back to pending
+					ticket.ticket_assignee = null
+					ticket.ticket_status = TICKET_PENDING
+					return claim
+				var/choice = tgui_alert(user, "This ticket has already been claimed by [assigned]! Do you wish to override their claim?", "Claim Override", list("Yes", "No"))
+				if(choice != "Yes")
+					claim = FALSE
+			if(claim)
+				ticket.ticket_assignee = MAIN_AI_SYSTEM
+				ticket.ticket_status = TICKET_ASSIGNED
+			return claim
+
+		if("auth_access")
+			var/datum/ares_ticket/access/access_ticket = locate(params["ticket"])
+			if(!access_ticket)
+				return FALSE
+			for(var/obj/item/card/id/identification in waiting_ids)
+				if(!istype(identification))
+					continue
```

Comment:
```suggestion
```
Since you didn't use `as anything` it should already be type checked in the for loop.